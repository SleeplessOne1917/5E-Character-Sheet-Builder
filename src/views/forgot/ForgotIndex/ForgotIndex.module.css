.header-text {
    color: var(--davy);
    font-family: var(--font-fantasy);
}

.button-container {
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    height: 13rem;
}

.content {
    display: flex;
    flex-direction: column;
    align-items: center;
}

@media screen and (min-width: 768px) {
    .header-text {
        font-size: 2.5rem;
    }
}

@media screen and (min-width: 1024px) {
    .header-text {
        font-size: 3rem;
    }
}