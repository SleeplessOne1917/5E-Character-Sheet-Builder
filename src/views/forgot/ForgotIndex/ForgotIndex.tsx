'use client';

import LinkButton from '../../../components/LinkButton/LinkButton';
import MainContent from '../../../components/MainContent/MainContent';
import classes from './ForgotIndex.module.css';

const ForgotIndex = () => {
	return (
		<MainContent testId="forgot-index">
			<div className={classes.content}>
				<h1 className={classes['header-text']}>Forgot Something?</h1>
				<div className={classes['button-container']}>
					<LinkButton href="/forgot/username">Forgot username</LinkButton>
					<LinkButton href="/forgot/password">Forgot password</LinkButton>
				</div>
			</div>
		</MainContent>
	);
};

export default ForgotIndex;
