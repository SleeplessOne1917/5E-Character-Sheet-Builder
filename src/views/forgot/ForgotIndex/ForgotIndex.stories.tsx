import { ComponentMeta, ComponentStory } from '@storybook/react';

import ForgotIndex from './ForgotIndex';

export default {
	title: 'Views/ForgotIndex',
	component: ForgotIndex
} as ComponentMeta<typeof ForgotIndex>;

const Template: ComponentStory<typeof ForgotIndex> = (args: any) => (
	<ForgotIndex {...args} />
);

export const Default = Template.bind({});
