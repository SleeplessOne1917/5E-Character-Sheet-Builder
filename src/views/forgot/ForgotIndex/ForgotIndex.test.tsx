import * as stories from './ForgotIndex.stories';

import { render, screen } from '@testing-library/react';

import { composeStories } from '@storybook/testing-react';

const { Default } = composeStories(stories);

it('renders correctly', () => {
	render(<Default />);

	expect(screen.getByTestId('forgot-index')).toMatchSnapshot();
});
