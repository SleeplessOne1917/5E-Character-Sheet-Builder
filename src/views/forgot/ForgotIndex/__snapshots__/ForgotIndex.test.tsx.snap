// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  class="main"
  data-testid="forgot-index"
>
  <div
    class="content"
  >
    <div
      class="content"
    >
      <h1
        class="header-text"
      >
        Forgot Something?
      </h1>
      <div
        class="button-container"
      >
        <a
          class="link-button"
          href="/forgot/username"
          tabindex="0"
        >
          Forgot username
        </a>
        <a
          class="link-button"
          href="/forgot/password"
          tabindex="0"
        >
          Forgot password
        </a>
      </div>
    </div>
  </div>
</main>
`;
